frappe/app.py

```diff
@@ -318,7 +321,10 @@ def handle_exception(e):
 	return response
 
 
-def after_request(rollback):
+def after_request(rollback: bool) -> bool:
```

Comment:
should `after_request` hooks be run in reverse order? :thinking: 

Kinda makes sense for monitoring-type hooks.

Current "hooks" in app.py follow it. 
![image](https://user-images.githubusercontent.com/9079960/218428142-5711d667-e20e-4e05-9717-76c8042c577b.png)

Reply:
@ankush 
> should after_requesst|job hooks be run in reverse order?

Makes sense but having the functions run in the defined order is easier to understand.

```
after_request = [
  'frappe.rate_limiter.update',
  'frappe.monitor.stop',
  'frappe.recorder.dump'
]
```

Reply:
With @netchampfaris on this. The reverse order bit can be maintained by the respective application authors. If Frappe does the reversing, it won't be intuitive.